woocommerce/woocommerce-android
# Issue/5332 use stripe extension account endpoint

Merge instructions:
1. Review this PR
2. Make sure https://github.com/woocommerce/woocommerce-android/pull/5375 is merged
3. Merge https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2204
4. Update fluxc hash in this PR
5. Remove "Do not merge" label
5. Merge this PR

Closes: #5332 (fluxc part)
Closes: #5334 (wcandroid part)


### Description


This PR updates WCAndroid so it's compatible with fluxC after introduction of /wc_stripe/account/summary endpoint - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2204

### Testing instructions

1. Install the newest version of Stripe Extension plugin to your store (the version hasn't be released yet, you'll need to build it locally)
2. Make sure WCPay is not active on your store
3. Open the App
4. Open app settings
5. Tap on "In-Person Payments"
6. Notice the account state is loaded - my site for example shows "Your WooCommerce Payments account has pending requirements" (I created [an issue](https://github.com/woocommerce/woocommerce-android/issues/5400) since it should not mention "WooCommerce Payments") )... you can also simply check what endpoint was invoked using flipper or similar tool

P.S. I haven't tested all the different states of the account yet, since we'll test that when the whole feature is ready.

### Images/gif

No changes
